### PR TITLE
Add proxy options

### PR DESCRIPTION
**Pull request by pandora2000**

### CODE REVIEW

**Comment by pandora2000** [comment]
@switschel Because setting environment variable like HTTP_PROXY is not working for `websocket-client`, I added proxy options. Please review. Documentation of proxy options is  https://websocket-client.readthedocs.io/en/latest/app.html#websocket._app.WebSocketApp.run_forever

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from setuptools import setup
 
 setup(name='c8y-device-proxy',
-      version='1.0.3',
+      version='1.1.0',
```
Comment (switschel) [inline setup.py:21]: I would suggest updating websocket_client to 1.2.1.
version 1.1.0 can be kept.

**Review by switschel — APPROVED** [review]
Fine for me. Will do some minor updates in main branch.

**Comment by pandora2000** [comment]
@switschel Thanks!

**Pull request merged**